### PR TITLE
feat: create CreatePlayone presentation and usecase

### PR DESCRIPTION
**Pull request by lekaha**

### CODE REVIEW

File: domain/src/main/java/com/playone/mobile/domain/model/Playone.kt
Comment (pokk) [inline domain/src/main/java/com/playone/mobile/domain/model/Playone.kt:33]: Why don't we use only one data class(Entity class)? Maintaining two class will cause inconsistent data class.

Comment (pokk) [reply domain/src/main/java/com/playone/mobile/domain/model/Playone.kt:33]: Also, our Entity Class will be a normal class, right? Do I miss something!?

File: presentation/src/main/java/com/playone/mobile/presentation/createPlayone/CreatePlayoneContract.kt
Comment (pokk) [inline presentation/src/main/java/com/playone/mobile/presentation/createPlayone/CreatePlayoneContract.kt]: The parameters become a class with all parameter will be awesome!

**Review by pokk — CHANGES_REQUESTED** [review]

File: domain/src/main/java/com/playone/mobile/domain/model/Playone.kt
Comment (lekaha) [reply domain/src/main/java/com/playone/mobile/domain/model/Playone.kt:33]: @pokk If use only one class it must have a field that is nullable i.e., `id` when creating Playone haven't had ID at the moment. That is why I made two class and one of two is not a entity class it is for parameters

File: presentation/src/main/java/com/playone/mobile/presentation/createPlayone/CreatePlayoneContract.kt
Comment (lekaha) [reply presentation/src/main/java/com/playone/mobile/presentation/createPlayone/CreatePlayoneContract.kt]: You think it is better to make another class for paramters?

Comment (pokk) [reply presentation/src/main/java/com/playone/mobile/presentation/createPlayone/CreatePlayoneContract.kt]: I think it's better as possible as we can. Once the parameters are changed, we need to change many many places(!?) at the same time.
If we keep the data class `data class CreateParameters`, it might be awesome to use it as a parameter here.

File: domain/src/main/java/com/playone/mobile/domain/model/Playone.kt
Comment (pokk) [reply domain/src/main/java/com/playone/mobile/domain/model/Playone.kt:33]: `limitPeople` should be in the `Detail` class I guess. 
We're able to avoid to as like kinda issues we missed some variables without the intent.

**Comment by pokk** [comment]
Please merge this after the #72 merges.

**Review by pokk — APPROVED** [review]

**Comment by pokk** [comment]
Just ignore the CI for now.

**Pull request merged**
